### ISSUE TITLE
Windows + Py3.5 packages fail

### ISSUE THREAD

**Issue by takluyver** [conda-forge/matplotlib-feedstock#47]
Running tests on Jenkins with Python 3.5 on Windows, we got errors like this:

```
  File "c:\jenkins\workspace\windows-multi\_pyversion\3.5\label\windows\IPython\core\tests\test_display.py", line 89, in test_set_matplotlib_formats
    from matplotlib.figure import Figure
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\figure.py", line 38, in <module>
    import matplotlib.colorbar as cbar
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\colorbar.py", line 34, in <module>
    import matplotlib.collections as collections
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\collections.py", line 27, in <module>
    import matplotlib.backend_bases as backend_bases
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\backend_bases.py", line 62, in <module>
    import matplotlib.textpath as textpath
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\textpath.py", line 15, in <module>
    import matplotlib.font_manager as font_manager
  File "C:\Users\ADMINI~1\AppData\Local\Temp\2\10111\lib\site-packages\matplotlib\font_manager.py", line 58, in <module>
    from matplotlib import ft2font
ImportError: DLL load failed: The specified module could not be found.
```

The errors disappear when we use the Continuum-maintained 'defaults' channel.

**Comment by pelson** [comment]
xref https://github.com/conda-forge/matplotlib-feedstock/issues/47

@jjhelmus - sorry to bother you, but do you see a similar thing on your Windows machine?

**Comment by jjhelmus** [comment]
@takluyver Can you provide some more details on how to replicate this error.  I tried installing python 3.5 from the `conda-forge` channel and numpy+matplotlib from `defaults` on a `win-32` VM and was not able to replicate this issue, `from matplotlib.figure import Figure` worked fine. 

It is possible this is related to conda-forge/python-feedstock#49 or conda-forge/python-feedstock#47

**Comment by takluyver** [comment]
Our Jenkins job does:

```
conda create -p %_envpath% python=%_pyversion% coverage jinja2 matplotlib mock nose numpy numpydoc pip pygments pyqt pywin32 pyzmq requests sphinx tornado
```

I think it's more likely due to the VC++ runtime. I saw similar issues with pyzmq (conda-forge/pyzmq-feedstock#9), which also uses C++. As I understand it (which is not thoroughly), Python depends on the Visual **C** runtime, but libraries written with C++ need an extra DLL. If you have that DLL already installed somewhere on your system, you likely won't be able to reproduce the error at all.

takluyver/pynsist#58 is where I learned about this.
